uui-components/src/widgets/Clickable.tsx

```diff
@@ -74,6 +78,11 @@ export const Clickable = React.forwardRef<HTMLButtonElement | HTMLAnchorElement
             }
 
             if (!!props.link) {
+                if (props.target) {
```

Comment:
For which case do we need it? It's a danger to add e.stopPropagation() component, because we are not sure that user don't have case to also catch this event somewhere else.

Reply:
We need it to add possibility open links in new tab using target = "_blank"